mrbgems/mruby-fiber/src/fiber.c

```diff
@@ -355,7 +355,6 @@ mrb_fiber_yield(mrb_state *mrb, mrb_int len, const mrb_value *a)
   if (c->vmexec) {
     c->vmexec = FALSE;
     mrb->c->ci->acc = CI_ACC_RESUMED;
-    c->cibase->pc = c->ci->pc;
```

Comment:
This line (saving the current execution point to `cibase->pc`) is an essential part of #5261 fix. Is it really OK to remove this line?

Reply:
Yes, it needs to be removed.
By this PR, in the case of child fiber, its `cibase->pc` already points to the address on `cibase->proc->body.irep->iseq`.